<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Initialization</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Port"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Baudrate"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>261</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Received packet from PLC</string>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Calibration start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Zero positioning start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Measuring start</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Abort measure</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Fault reset</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Save Measurements</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>New Data Transfer</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_scanLength">
       <property name="text">
        <string>scan length: </string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_calibDiam">
       <property name="text">
        <string>calid deameter: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>501</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Packet for sending to PLC</string>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>State</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_calibration_running">
          <property name="text">
           <string>calibration_running</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_calibration_completed">
          <property name="text">
           <string>calibration_completed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_calibration_error">
          <property name="text">
           <string>calibration_error</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_zero_positioning_running">
          <property name="text">
           <string>zero_positioning_running</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_zero_positioning_completed">
          <property name="text">
           <string>zero_positioning_completed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_zero_positioning_error">
          <property name="text">
           <string>zero_positioning_error</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_measuring_running">
          <property name="text">
           <string>measuring_running</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_measuring_error">
          <property name="text">
           <string>measuring_error</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_measuring_completed">
          <property name="text">
           <string>measuring_completed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_measuring_aborted">
          <property name="text">
           <string>measuring_aborted</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_system_in_fault_condition">
          <property name="text">
           <string>system_in_fault_condition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_save_measurements_running">
          <property name="text">
           <string>save_measurements_running</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_save_measurements_completed">
          <property name="text">
           <string>save_measurements_completed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_save_measurements_error">
          <property name="text">
           <string>save_measurements_error</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_data_transfer_read">
          <property name="text">
           <string>data_transfer_read</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_diameter_min"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>diameter_min</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_diameter_max"/>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineEdit_diameter_avg"/>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit_chamfer_width_min"/>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="lineEdit_chamfer_width_max"/>
        </item>
        <item row="5" column="0">
         <widget class="QLineEdit" name="lineEdit_chamfer_width_avg"/>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="lineEdit_circularity_min"/>
        </item>
        <item row="7" column="0">
         <widget class="QLineEdit" name="lineEdit_circularity_max"/>
        </item>
        <item row="8" column="0">
         <widget class="QLineEdit" name="lineEdit_circularity_avg"/>
        </item>
        <item row="9" column="0">
         <widget class="QLineEdit" name="lineEdit_bevel_angle_min"/>
        </item>
        <item row="10" column="0">
         <widget class="QLineEdit" name="lineEdit_bevel_angle_max"/>
        </item>
        <item row="11" column="0">
         <widget class="QLineEdit" name="lineEdit_bevel_angle_avg"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>diameter_max</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>diameter_avg</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>chamfer_width_min</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>chamfer_width_max</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>chamfer_width_avg</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>circularity_min</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>circularity_max</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>circularity_avg</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>bevel_angle_min</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>bevel_angle_max</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>bevel_angle_avg</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLineEdit" name="lineEdit_band_height_min"/>
        </item>
        <item row="13" column="0">
         <widget class="QLineEdit" name="lineEdit_band_height_max"/>
        </item>
        <item row="14" column="0">
         <widget class="QLineEdit" name="lineEdit_band_height_avg"/>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>band_height_min</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>band_height_max</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>band_height_avg</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_sendPacket">
       <property name="text">
        <string>Send Packet</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>771</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
